.github/workflows/ansible-test.yml

```
@@ -37,12 +37,19 @@ jobs:
         ansible:
           # It's important that Sanity is tested against all stable-X.Y branches
           # Testing against `devel` may fail as new tests are added.
+          # An alternative to `devel` is the `milestone` branch with
+          # gets synchronized with `devel` every few weeks and therefore
+          # tends to be a more stable target. Be aware that it is not updated
+          # around creation of a new stable branch, this might cause a problem
+          # that two different versions of ansible-test use the same sanity test
+          # ignore.txt file.
         # - stable-2.9 # Only if your collection supports Ansible 2.9
           - stable-2.10
```

While we're at it, should we also change this to
```suggestion
        # - stable-2.10 # Only if your collection supports ansible-base 2.10
```
? After all, that's also EOL by now. (Same below.)

---

I don't like mixing unrelated things in the same PR.

---

But I've made a separate one: https://github.com/ansible-collections/collection_template/pull/49.